numba/core/extending.py

```diff
@@ -57,7 +57,7 @@ def generic(self):
 
 
 def overload(func, jit_options={}, strict=True, inline='never',
-             prefer_literal=False):
+             prefer_literal=False, **kwargs):
```

Comment:
Can we leave a note about `hardware` being the only accepted key for kwargs?

Reply:
Agree, done.